internal/config/mount_config.go

```diff
@@ -24,9 +24,29 @@ type LogConfig struct {
 	FilePath string      `yaml:"file-path"`
 }
 
+type FileCacheConfig struct {
+	Path                    string `yaml:"path"`
+	Size                    int64  `yaml:"size-mb"`
+	TTL                     int64  `yaml:"ttl-sec"`
+	CacheFileForRandomReads bool   `yaml:"cache-file-for-random-reads"`
+}
+
+type MetadataCacheConfig struct {
+	Capacity int64 `yaml:"capacity"`
+	TTL      int64 `yaml:"ttl-sec"`
+}
+
+type TypeCacheConfig struct {
+	TTL              int64 `yaml:"ttl-sec"`
+	CacheNonexistent bool  `yaml:"cache-nonexistent"`
+}
+
 type MountConfig struct {
-	WriteConfig `yaml:"write"`
-	LogConfig   `yaml:"logging"`
+	WriteConfig         `yaml:"write"`
+	LogConfig           `yaml:"logging"`
+	FileCacheConfig     `yaml:"file-cache"`
```

Comment:
can you check what is preferred in yaml underscore or hypen?

Reply:
As discussed offline, this is dependent on the language around which the yaml is used. And if we consider that, the naming should be like jobName. However, that will not be suitable as all our flags are like job-name. So, we will go with hypens job-name. Also, we will go with hyphens at all levels in config file i.e. 
config.yaml

- first-level:
  - second-level: some_value

The above makes things consistent.